src/components/dashboard/SendTokens/lang.js

```diff
@@ -8,6 +8,7 @@ export const prefix = 'SendTokens'
 export default {
   en: {
     fast: 'Fast',
+    instantSend: 'InstantSend',
```

Comment:
Should we use space between words? Like "Instant send"

Reply:
I think no. It is used as a single word in camel case (see e.g. 
https://docs.dash.org/en/latest/wallets/dashcore/privatesend-instantsend.htm)l